frappe/api/v2.py

```diff
@@ -114,6 +114,51 @@ def delete_doc(doctype: str, name: str):
 	frappe.response.http_status_code = 202
 	return "ok"
 
+def get_attachments(doctype: str, name: str):
+	from frappe.desk.form.load import get_attachments
+	return get_attachments(doctype, name)
+
+def add_attachment(doctype: str, name: str):
+	frappe.form_dict.update({
+		"doctype": doctype, 
+		"docname": name
+	})
+	return upload_file()
+
+def remove_attachment():
+	from frappe.desk.form.utils import remove_attach
+	return remove_attach()
+
+def get_tags(doctype: str, name: str):
+	from frappe.desk.form.load import get_tags
+	return get_tags(doctype, name)
+
+def add_tag(doctype: str, name: str):
+	from frappe.desk.doctype.tag.tag import add_tag
+	return add_tag(tag=frappe.form_dict.tag, dt=doctype, dn=name)
+
+def remove_tag(doctype: str, name: str):
+	from frappe.desk.doctype.tag.tag import remove_tag
+	return remove_tag(tag=frappe.form_dict.tag, dt=doctype, dn=name)
+
+def get_assignments(doctype: str, name: str):
+	from frappe.desk.form.assign_to import get as getAssignments
```

Comment:
```suggestion
	from frappe.desk.form.assign_to import get as get_assignments
```
And similar for the other 2, was there any specific reason for this naming?

Reply:
no reason, i didnt notice wrong case.. will fix such cases